vm/isolated/isolated.go

```diff
@@ -65,6 +71,7 @@ func ctor(env *vmimpl.Env) (vmimpl.Pool, error) {
 	if env.Debug && len(cfg.Targets) > 1 {
 		log.Logf(0, "limiting number of targets from %v to 1 in debug mode", len(cfg.Targets))
 		cfg.Targets = cfg.Targets[:1]
+		cfg.USBDevNums = cfg.USBDevNums[:1]
```

Comment:
Looks like this will fail when `cfg.USBDevNums` is empty. I think it actually makes sense to always check that `cfg.USBDevNums` length equals `cfg.Targets` length if the former is > 0.